DNM: add migrators for CTK12/13

billysuh7 opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

This is just a test for rerendering

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

billysuh7 committed:
add migrators

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/17165610891](https://github.com/conda-forge/conda-forge-webservices/actions/runs/17165610891). Examine the logs at this URL for more detail.</sub>

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you but ran into some issues. Please check the [output logs](https://github.com/conda-forge/conda-forge-webservices/actions/runs/17165617631) of the GitHub Actions workflow below for more details. You can also ping `conda-forge/core` (using the `@` notation) for further assistance or you can try [rerendering locally](https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-with-conda-smithy-locally).


The following suggestions might help debug any issues:
* Is the `recipe/{meta.yaml,recipe.yaml}` file valid?
* If there is a `recipe/conda-build-config.yaml` file in the feedstock make sure that it is compatible with the current [global pinnnings](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml).
* Is the fork used for this PR on an organization or user GitHub account? Automated rerendering via the webservices admin bot only works for user GitHub accounts.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/17165617631](https://github.com/conda-forge/conda-forge-webservices/actions/runs/17165617631). Examine the logs at this URL for more detail.</sub>

jakirkham committed:
Remove custom `conda_build_config.yaml`

jakirkham committed:
Vendor CUDA 12.9 migrator to use GLIBC 2.28

The default GLIBC version with the CUDA 12.9 migrator is GLIBC 2.17.
However this package requires GLIBC 2.28.

Given this, vendor the migrator in the feedstock by setting
`use_local: true`. This way the feedstock's copy of this migrator is
used in favor of the global migrator. Additionally change the
`c_stdlib_version` to `2.28`. Without `use_local: true`, this last
change would not work. However together this will override the GLIBC
version used in the feedstock.

Commit:
MNT: Re-rendered with conda-build 25.7.0, conda-smithy 3.51.1, and conda-forge-pinning 2025.08.22.18.58.01

Commit:
MNT: Re-rendered with conda-build 25.7.0, conda-smithy 3.51.1, and conda-forge-pinning 2025.08.22.18.58.01

jakirkham reviewed (commented):
Thanks for working on this Billy! 🙏

Based on our discussion offline, I thought it would help if I pushed a couple changes here and added a few comments on the diff to explain them. Hope that is ok

When we have a case where we want to change some migrator contents, we need to vendor the migrator in the feedstock. We do this by adding `use_local: true`. By doing this we are able to make other changes to the migrator and `conda-smithy` will pick them up during re-rendering

Have provided more detail about these changes and how they work below. Please let me know if you have any questions 🙂

jakirkham commented:
Thanks again for working on this Billy! 🙏

It looks like this was done in PR: https://github.com/conda-forge/libnvshmem-feedstock/pull/13

Do we still need changes from this PR or is it good to close?
